charts/rancher-backup/values.yaml

```diff
@@ -90,3 +90,19 @@ monitoring:
     additionalLabels: {}
     metricRelabelings: []
     relabelings: []
+  
+  alerts:
+    enabled: false
+
+    ## Define custom alerting rules here.
+    ## The "BackupFailed" alert is included by default when .Values.monitoring.alerts.enabled is set to true and rancher-monitoring is installed.
+    alertingRules:
```

Comment:
I would also suggest putting this default alert rule definition in helm template behind a template flag like .Values.useDefaultAlertRules or something and the current definition should be behind `.Values.additionalRules`